default.build

```diff
@@ -39,6 +40,8 @@
 			<arg value="/p:IncludeSymbols=&quot;True&quot;" />
 			<arg value="/p:IncludeSource=&quot;True&quot;" />
 			<arg value="/p:PackageOutputPath=&quot;${path::get-full-path(nuget.nupackages.dir)}&quot;" />
+			<!-- build.counter is not available by default. It needs to be exposed in Team City as system parameter system.build.counter with value %build.counter% -->
+			<arg value="/p:BuildNumber=&quot;${build.counter}&quot;" />
```

Comment:
I think it's better to add `if="${property::exists('build.counter')}"` element

Reply:
It doesn't work. I've removed it in my last commit. See https://github.com/nhibernate/nhibernate-core/pull/2254#issuecomment-544115119

Reply:
I added skip if it's empty